Corona-Warn-App/src/main/res/drawable-night/register_test_card_background.xml

```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="336dp"
+    android:height="243dp"
+    android:viewportWidth="336"
+    android:viewportHeight="243">
+    <path android:pathData="M0,4C0,1.79 1.79,0 4,0H332C334.21,0 336,1.79 336,4V243H0V4Z">
+        <aapt:attr name="android:fillColor">
```

Comment:
I think this will cause a crash on Android 6, `fillColor` is supported from API 24. could you please double check it?

Reply:
So how should I import  them then? I've used the same method up until now.

Reply:
I remember you used a tool to reduce the size of these drawables. Could that actually remove the fillColor completely?

Reply:
You need to ask Anja to export it differently, the tool won't fix it

Reply:
I did a quick check to see if this is used anywhere else and it's found in many drawables. I guess if it caused a crash it would have been reported by now.

Reply:
Ok, please check it on Android 6 if it works ?

Reply:
Ok, it doesn't work when using the resource to set the background but it works fine for the icon... I'll ask Anja

Reply:
UX was unable to produce an asset without this property but I pushed a workaround where instead of setting the drawable as the background of the `ConstraintLayout` I simply created an `ImageView` and set it to span accross the whole top part of the card with `srcCompat` and that seems to prevent the app from crashing on 6.0

Reply:
works